DesignPatterns/TemplateMethod/Kata/TemplateMethodKata.md

```diff
@@ -0,0 +1,74 @@
+# Template Method Kata
+
+Before attempting the kata, read the article, the article code, and article tests.
```

Comment:
Pointing this out again. should read: "read the article, article code, and article tests". Update in all PR's please.